### PR TITLE
included more thorough explanations of files/directories in .gitignore

### PR DESCRIPTION
**Pull request by Jtiv**
resolving ticket #12

### CODE REVIEW

**Comment by Jtiv** [comment]
resolving issues #12

File: .gitignore [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
 /build
 
-# misc
+# .DS_Store gets generated by macOS users when viewing files in Finder. Not needed.
+# These are environment files that should not be included remotely for security.
 .DS_Store
 .env.local
 .env.development.local
 .env.test.local
 .env.production.local
```
Comment (Underwaterr) [inline .gitignore:21]: Let's ditch the extra `.env.whatever` files and only have `.env`. Make the DS_Store comment only above the DS_Store rule, and same for env comment and rule

**Review by Underwaterr — CHANGES_REQUESTED** [review]

**Comment by Jtiv** [comment]
Rerequested resolvment of issue #12

**Review by Underwaterr — APPROVED** [review]
Woo yeah party time excellent

**Pull request merged**